CHANGELOG.md

```diff
@@ -1,24 +1,11 @@
-## [3.5.1](https://github.com/algolia/instantsearch.js/compare/v3.5.0...v3.5.1) (2019-05-20)
+## [3.5.1](https://github.com/algolia/instantsearch.js/compare/v3.4.0...v3.5.1) (2019-05-20)
```

Comment:
We can make it a `h1` since this is the first minor version of `3.5.x`.

```suggestion
# [3.5.1](https://github.com/algolia/instantsearch.js/compare/v3.4.0...v3.5.1) (2019-05-20)
```

Reply:
I hope this doesn't break next time. 🙏